Packs/Securonix/Integrations/Securonix/Securonix.py

```diff
@@ -1135,23 +1136,31 @@ def fetch_incidents(client: Client, fetch_time: Optional[str], incident_status:
     securonix_incidents = client.list_incidents_request(from_epoch, to_epoch, incident_status)
 
     if securonix_incidents:
+        # Securonix returns the recent 10 incidents, offsetting the response to get the oldest 10 incidents
+        total_incidents = int(securonix_incidents.get('totalIncidents', 0))
+        if total_incidents > 10:
+            demisto.info(f'Fetching Securonix incidents. With offset: {total_incidents-10}')
+            securonix_incidents = client.list_incidents_request(from_epoch, to_epoch, incident_status,
+                                                                int(total_incidents - 10))
+
         incidents_items = list(securonix_incidents.get('incidentItems'))  # type: ignore
-        last_incident_id = last_run.get('id', '0')
+        # 1st incident is the latest
+        if incidents_items:
+            now = timestamp_to_datestring(incidents_items[0].get('lastUpdateDate'))
```

Comment:
1) is lastUpdateDate is the correct value to use? shouldn't it be creation time?
2)you are using the exact time dose the query in Securonix brings every thing that is ">= now"
or "> now"?

Reply:
Asked the vendor both, still waiting for reply.

Reply:
See comments in the referenced issue, there are issues with the current way listing the incidents was implemented by the vendor.